framework/src/problems/FEProblemBase.C

```diff
@@ -2324,6 +2325,32 @@ FEProblemBase::getMaterial(std::string name,
     case Moose::FACE_MATERIAL_DATA:
       name += "_face";
       break;
+    // case Moose::INTERFACE_MATERIAL_DATA:
```

Comment:
this I'm not sure what to do as @andrsd wrote in the previous PR to get two different methods. but who call this??